plugin.video.zoneminder/resources/settings.xml

```diff
@@ -0,0 +1,12 @@
+<settings>
```

Comment:
It'd be nice if you could update the settings to the new format. You could use https://gitlab.com/ronie/script.settings-convert for an automatic conversion